hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHomeMetrics.java

```diff
@@ -133,7 +133,18 @@
      *   D means it runs in docker but not on kubernetes ,
      *   N means it doesn't run in docker
      */
-    DOCKER("dck");
+    DOCKER("dck"),
+
+    //JET METRICS
+    JET_ENABLED("jet"),
+    JET_RESOURCE_UPLOAD_ENABLED("jetrsup"),
+    JET_JOBS_SUBMITTED("jetjobss"),
```

Comment:
Regarding naming, do we want to keep adding "jet" in names, or shall we use "streaming" in new code? Can you please confirm it with core developers?

Reply:
Jet is fine, it's not going away any time soon.